qml/main.qml

```diff
@@ -46,9 +46,27 @@ Window {
 
         Loader {
             id: spotShapeLoader
+            anchors.centerIn: centerRect
             width: centerRect.width;  height: width
             sourceComponent: Qt.createComponent(Settings.spotShape)
         }
+        Loader {
+            id: spotShapeLoader2
+            objectName: "spotarea"
+            anchors.centerIn: centerRect
+            width: centerRect.width;  height: width
+            sourceComponent: Qt.createComponent(Settings.spotShape)
+            visible: Settings.shoSpot || Settings.showBorder
```

Comment:
Typo: shoSpot

Reply:
It should be `Settings.showSpot && Settings.showBorder`

Reply:
Actually I like the idea of showing the border without the spot also. So we could just use `visible: Settings.showBorder` - what do you think?

Reply:
Yes, With other option combination (like dot in center being active and shade opacity kept to zero) a nice effect is obtained by having border.